lib/config/index.js

```diff
@@ -157,7 +157,7 @@ for (let i = keys.length; i--;) {
 
 // Notify users about the prefix change and inform them they use legacy prefix for environment variables
 if (Object.keys(process.env).toString().indexOf('HMD_') !== -1) {
-  logger.warn('Using legacy HMD prefix for environment variables. Please change your variables in future. For details see: https://github.com/codimd/server#environment-variables-will-overwrite-other-server-configs')
+  logger.warn('Using legacy HMD prefix for environment variables. Please change your variables in future. For details see: https://github.com/hedgedoc/hedgedoc#environment-variables-will-overwrite-other-server-configs')
```

Comment:
This link is dead, as the relevant section is not part of the README anymore.

Reply:
I changed the link to the configuration doc